### DISCUSSION TITLE
需要一份詞庫匯總

### DISCUSSION THREAD

**Discussion by royjor2016** [rime/rime-cantonese#175] [Q&A] [answered]
有時打字見到某d詞 覺得佢寫錯 或者直頭唔係粵語詞 想改 但係唔知個詞來自邊份yaml

唔該列晒rime canto會引用嘅詞庫出來，好 mou2 ？

譬如我想搵 髀樞 喺邊份yaml入邊。我想刪咗佢。依家無從落手。

**Comment by laubonghaudoi** [comment, marked as answer]
https://github.com/CanCLID/rime-cantonese-upstream

**Reply by royjor2016** [reply]
> rime-cantonese 作為下游輸入法碼表會通過 CI 自動從本倉庫揸取更新構建新碼表。
請問咩係 CI ？

**Reply by laubonghaudoi** [reply]
https://github.com/rime/rime-cantonese/blob/main/.github/workflows/fetch-upstream.yml

**Reply by royjor2016** [reply]
啱啱搵咗下 搵唔到 髀樞 出自任何一份csv

**Reply by laubonghaudoi** [reply]
有可能係你以前唔小心打咗呢個詞跟住入咗個用户詞庫記憶。

**Reply by royjor2016** [reply]
梗唔係laa 我爲咗打秘書先見到呢個詞

**Reply by laubonghaudoi** [reply]
亦有可能係八股文自帶詞彙 https://github.com/rime/rime-essay

**Reply by royjor2016** [reply]
係laa 見到laa
請問點樣disable佢呢？我唔想要亦唔需要rime essay。

**Reply by royjor2016** [reply]
其實有冇得整一份淨係得單字嘅rime canto
我估靠我自己打，佢自己學，噉來build up個詞表，仲好過依家喺成zaa狗屎垃圾之中揀。no offence but成堆亂七八糟嘅提示真係好煩。

**Reply by laubonghaudoi** [reply]
建議你去rime提問，呢個唔關 rime-cantonese事，或者去telegram羣問睇有冇人會幫你
